setup.cfg

```diff
@@ -40,7 +40,7 @@ include_package_data = True
 install_requires =
     pandas >= 1.3.0, <1.4.2
     scikit-learn >= 0.22
-    pyarrow >= 1.0.0
+    pyarrow >= 5.0.0
```

Comment:
Pyarrow versions less than this don't include the `pyarrow._orc` module that is required for the way ww writes or reads files. We raise the limit of the pyarrow version to satisfy this.

Reply:
Nice